core/src/main/java/org/apache/iceberg/DataFilesTable.java

```diff
@@ -142,5 +154,10 @@ public long targetSplitSize() {
     public Iterable<FileScanTask> split(long splitSize) {
       return ImmutableList.of(this); // don't split
     }
+
+    @VisibleForTesting
+    ManifestFile getManifest() {
```

Comment:
I realize this is only for testing, but I would still suggest using a method name that we don't have to change later if we want to use the method elsewhere. That means removing `get` because `get` doesn't add anything. Typically, `get` signals that the method name can be simpler (e.g., `manifest` to return the manifest) or should have a more descriptive verb like `find`, `fetch`, `create`, etc. that tell the caller more about what is happening. The only time we use `get` is when the object needs to be a Java bean.

Reply:
Ah got it , changed